### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by timholy** [JuliaImages/Images.jl#1]
Does this have anything new in it? I am going to keep fixing bugs over at Images.jl. I was thinking of forking on C-day, just to make sure everything stays well-aligned. But if you want to merge the new patches over here, that's fine too.

**Comment by SimonDanisch** [comment]
No, nothing new here! I just was about to get started, but I could also start with other things first.

**Comment by timholy** [comment]
OK, let's delete this for now---merging between repos gets more confusing than merging between branches.

I just made you a collaborator over at timholy/Images.jl, so you can push directly without having to submit pull requests (unless you want me to review it for you). You can start a branch based off `teh/colors`.
